src/logic/function/basic.lean

```diff
@@ -348,6 +348,36 @@ by {funext b, by_cases b = a; simp [update, h]}
 
 end update
 
+namespace injective
+
+noncomputable theory
+local attribute [instance, priority 10] classical.prop_decidable
+
+variables {α β γ : Type*} {f : α → β} (hf : injective f)
+
+/-- `extend hf g e'` extends a function `g : α → γ`
+along an injective (`hf`) function `f : α → β`
+to a function `β → γ`, by using the values of of `g` on the range of `f`
+and the values of an auxiliary function `e' : β → γ` elsewhere. -/
+def extend (hf : injective f) (g : α → γ) (e' : β → γ) : β → γ :=
```

Comment:
This will show as `extend _ g e'` in the pretty printer otherwise.
```suggestion
def extend (f : α → β) (hf : injective f) (g : α → γ) (e' : β → γ) : β → γ :=
```

Reply:
Fixed